### ISSUE TITLE
Optional Postgres/Redis for KubeApps Helm Chart

### ISSUE THREAD

**Issue by drew-viles** [vmware-tanzu/kubeapps#6596]
**Summary**
To prevent multiple workloads of the same type being deployed into a management cluster, it'd be nice if things like the Redis and Postgresql values were able to be pointed to an existing configuration, not just disabled.

I see that Postgres can be disabled, Redis cannot but if I do disable, them there is no way of providing an existing setup to use.

For example, I already have a postgresql-ha bitnami workload running which manages the databases for some general management workloads in my cluster, dex, harbor and a few other workloads, and ideally I'd like to point kubeapps at these installations, not setup another non-ha deployment.

The same goes for Redis. I already have a HA Redis workload running and would like to make use of this instead of installing another identical copy of what I already have in the cluster.

**Background and rationale**
Prevents the duplication of workloads - saves resources.

**Description**
I can't really add any more than I have, so here is a dog to help convince you 🐶 

**Acceptance criteria**
The ability to define an "external" Postgresql and Redis rather than having to use the ones bundled in with the chart.

**Other Information**
I originally opened this in the [bitnami helm charts](https://github.com/bitnami/charts/issues/18281) repo but they pointed me here.

**Comment by absoludity** [comment]
> For example, I already have a postgresql-ha bitnami workload running which manages the databases for some general management workloads in my cluster, dex, harbor and a few other workloads, and ideally I'd like to point kubeapps at these installations, not setup another non-ha deployment.

Yep, makes sense. Note that the postgres and redis data used by Kubeapps does not require HA as it is a cache that gets rebuilt when needed. But that doesn't invalidate your point about resource usage.

> **Description** I can't really add any more than I have, so here is a dog to help convince you 🐶

:P

**Comment by jdenson** [comment]
just here to show my support for this suggestion 👍

**Comment by smithatlanta** [comment]
Me, too

**Comment by parviz-miriyev** [comment]
yep, same here, better to have choice for external postgres and redis
